comp/core/configsync/configsyncimpl/sync.go

```diff
@@ -18,7 +18,12 @@ import (
 )
 
 func (cs *configSync) updater() {
-	cfg, err := fetchConfig(cs.ctx, cs.client, cs.Authtoken.Get(), cs.url.String())
+	authToken, err := cs.Authtoken.Get()
+	if err != nil {
+		cs.Log.Errorf("Failed to fetch config from core agent: unable to retrieve auth_token: %v", err.Error())
+	}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I think this should return ? It won't be able to fetch anything if it doesn't have the auth token anyway
```suggestion
	authToken, err := cs.Authtoken.Get()
	if err != nil {
		cs.Log.Errorf("Failed to fetch config from core agent: unable to retrieve auth_token: %v", err.Error())
		return
	}
```

Reply:
You're right !